cocotb/share/lib/gpi_log/gpi_logging.c

```diff
@@ -173,15 +179,14 @@ void gpi_log(const char *name, long level, const char *pathname, const char *fun
     PyTuple_SetItem(call_args, 3, PyUnicode_FromString(log_buff));   // Note: This function steals a reference.
     PyTuple_SetItem(call_args, 4, PyUnicode_FromString(funcname));
 
-    retuple = PyObject_CallObject(pLogHandler, call_args);
-
-    if (retuple != Py_True) {
```

Comment:
`handler` doesn't return anything (aka returns `Py_None`), so all that needs to be checked is whether an exception occured.